KeysPerSecond/src/me/roan/kps/Main.java

```diff
@@ -272,31 +279,32 @@ protected static final void mainLoop(){
 		future = timer.scheduleAtFixedRate(new Runnable() {
 			@Override
 			public void run() {

```

Comment:
Not sure why, but for me nothing works anymore. The program still keeps track of hits/key. But nothing else works. I verified and this method is called, but nothing happens.